webpack.server.config.js

```diff
@@ -70,8 +70,11 @@ module.exports = {
   },
 
   plugins: [
+    new webpack.NormalModuleReplacementPlugin(
+      /jquery/,
+      path.resolve(dirs.SCRIPTS, 'empty.js'),
+    ),
```

Comment:
You're using `IgnorePlugin` for `@popperjs` in https://github.com/metabrainz/musicbrainz-server/pull/1744 - should this approach also be used there then?